src/App.jsx

```diff
@@ -14,70 +15,66 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [food, setFood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      {food.length !== 0 ? (
+        /* eslint-disable */
+        <h1 className="title is-flex is-align-items-center">
+          {`${food} is selected`}
```

Comment:
I think you can write here just
`{food ? (
        /* eslint-disable */
        <h1 className="title is-flex is-align-items-center">`